QUEST_LV_0100.tsv

```diff
@@ -3852,28 +3856,28 @@ QUEST_LV_0100_20150317_003851	Report to Knight Titas.
 QUEST_LV_0100_20150317_003853	$Find Search Scout in the marked area
 QUEST_LV_0100_20150317_003854	$You were asked to inform the troops to assemble before heading to Klaipeda. Inform the Search Scout about the assembly.
-QUEST_LV_0100_20150317_003855	$Inform the Search Scout about the shift.
+QUEST_LV_0100_20150317_003855	$Inform the Search Scout about the order.
 QUEST_LV_0100_20150317_003856	$A Large Kepa suddenly appeared. Defeat it.
 QUEST_LV_0100_20150317_003857	$Talk to Search Scout
 QUEST_LV_0100_20150317_003858	$Inform the Search Scout about the order to assemble and find the location of the next soldier.
 QUEST_LV_0100_20150317_003859	$Defeat the Large Kepa
 QUEST_LV_0100_20150317_003860	$Talk to Laimonas
-QUEST_LV_0100_20150317_003861	$Let's listen to Laimonas' favor like Titas said.
-QUEST_LV_0100_20150317_003862	$Offer a prayer to the Goddess Statue.
-QUEST_LV_0100_20150317_003863	$Offer a prayer to the Statue of Goddess Zemyna. The statue is located at the end of the road on the right of Laimonas.
+QUEST_LV_0100_20150317_003861	$Let's listen to Laimonas' favor like Sir Titas said.
+QUEST_LV_0100_20150317_003862	$Worship the Goddess Statue.
+QUEST_LV_0100_20150317_003863	$Worship the Statue of Goddess Zemyna. The statue is located at the end of the road on the right of Laimonas.
 QUEST_LV_0100_20150317_003864	$You offered a prayer as Laimonas asked you to. Return and tell Laimonas about it.
-QUEST_LV_0100_20150317_003865	$Listen to Laimonas' favor.
-QUEST_LV_0100_20150317_003866	$Defeat the Infrorocktor near the marked area.
+QUEST_LV_0100_20150317_003865	$Listen to Laimonas' request.
+QUEST_LV_0100_20150317_003866	$Defeat Infrorocktors at the marked area.
 QUEST_LV_0100_20150317_003867	$Do Laimonas a favor and defeat the Infrorocktors on the road to Klaipeda.
 QUEST_LV_0100_20150317_003868	Talk to the Klaipeda Guard Captain
-QUEST_LV_0100_20150317_003869	You seem to have taken care of Laimonas's favor. Tell the Guard Captain that you got rid of the monsters on the road to Klaipeda.
-QUEST_LV_0100_20150317_003870	$Defeat Infrorocktor
+QUEST_LV_0100_20150317_003869	You seem to have taken care of Laimonas's request. Tell the Guard Captain that you got rid of the monsters on the road to Klaipeda.
+QUEST_LV_0100_20150317_003870	$Defeat Infrorocktors
 QUEST_LV_0100_20150317_003871	$Go to Klaipeda
-QUEST_LV_0100_20150317_003872	$Let's move on the way to the Klaipeda.
+QUEST_LV_0100_20150317_003872	$Let's go to Klaipeda.
 QUEST_LV_0100_20150317_003873	$Defeat the Rocktortuga that suddenly appeared.
-QUEST_LV_0100_20150317_003874	$Defeat the Rocktortuga that appeared suddenly before entering the Klaipeda.
+QUEST_LV_0100_20150317_003874	$Defeat the Rocktortuga that suddenly appeared before the entrance to Klaipeda.
 QUEST_LV_0100_20150317_003875	$Report to the Guard Captain on Rocktortuga's defeat.
-QUEST_LV_0100_20150317_003876	Pass the orders to the Battle Commander
+QUEST_LV_0100_20150317_003876	Pass the Orders to the Battle Commander
```

Comment:
Pass the Orders to the Battle Commander -> Pass the orders to the Battle Commander